core/src/test/scala/io/qbeast/core/transform/TransformerTest.scala

```diff
@@ -40,6 +42,28 @@ class TransformerTest extends AnyFlatSpec with Matchers {
     }
   }
 
+  it should "makeTransformation with Timestamp data type" in {
+    val columnName = "a"
+    val dataType = TimestampDataType
+    val transformer = Transformer(columnName, dataType)
+
+    val minTimestamp = Timestamp.valueOf("2017-01-01 12:02:00")
+    val maxTimestamp = Timestamp.valueOf("2017-01-02 12:02:00")
+
+    val minTimestampLong = minTimestamp.getTime
+    val maxTimestampLong = maxTimestamp.getTime
+
+    // If I do not print the values it complains that variables are not used
+    Console.print(minTimestampLong)
```

Comment:
This should not be the case, since you are using them below. Maybe was something related to sbt configuration?

Reply:
I don't know why this happens, I think it is related to the variables being used inside the matchPattern. If I remove the print I get this: 
`local val minTimestampLong in value <local TransformerTest> is never used
 val minTimestampLong = minTimestamp.getTime`

Reply:
Instead of asserting with matchPattern, try with the clause` shouldBe`. I don't know much about the matchPattern behaviour.

Reply:
I changed the test implementation, It feels way more complicated than it should because the object does not have methods implemented to get the parameters of the class. It works but I would change it if we figure out a better way to do it.